projects/observability/src/shared/components/top-n/top-n-chart.component.ts

```diff
@@ -1,30 +1,18 @@
 import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
 import { TypedSimpleChanges } from '@hypertrace/common';
-import { maxBy } from 'lodash-es';
+import { GaugeData } from '../gauge/gauge-chart.component';
 
 @Component({
   selector: 'ht-top-n-chart',
   styleUrls: ['./top-n-chart.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div class="top-n-chart">
-      <ng-container *ngFor="let item of this.itemOptions">
-        <div class="border-top"></div>
-        <div
-          class="label"
-          [htcTooltip]="item.label"
-          (click)="this.onItemClick(item)"
-          [ngClass]="{ clickable: this.labelClickable }"
-        >
-          {{ item.label }}
-        </div>
-        <div class="progress">
-          <div class="progress-value" [ngStyle]="{ width: item.width }"></div>
-        </div>
-        <div class="value">
-          <span>{{ item.value | htcDisplayNumber }}</span>
-        </div>
-      </ng-container>
+      <ht-gauge-chart
```

Comment:
If all this is doing is sorting, my vote would be to get rid of this component. Use the underlying, unsorted chart directly from the renderer and have the renderer do the sorting. It seems excessive to have an extra component layer to go `widget -> sorted -> unsorted`